packages/target-decisioning-engine/src/utils.js

```diff
@@ -262,10 +196,14 @@ export function determineArtifactLocation(config) {
     return artifactLocation;
   }
 
-  const targetEnvironment = getTargetEnvironment(config);
+  const cdnBaseUrl = `https://${CDN_BASE_PATH}`;
+  const targetEnvironment =
+    config.environment != null
```

Comment:
You can remove the `!= null`.  That way if it is null, undefined, or empty string it will use the prod env.

Reply:
truthy